frontend/src/components/news-page/NewsCard.scss

```
@@ -0,0 +1,11 @@
+// .video-container {
+//   width: 100%;
+//   aspect-ratio: 1 / 1;
+//   overflow: hidden;
+//   display: block;
+// }
+
+video {
```

This will style all videos on the site - I know there aren't going to be any others, but could you either give this an extra class e.g:
```suggestion
video.reddit-whale-video {
```
or style it only while it's inside a higher-level component, e.g:
```suggestion
.news-page {
  video {
```